cypress/test/search-toggle-facet.spec.ts

```diff
@@ -14,12 +14,9 @@ describe('Search Togle facet', () => {
     // When data is loaded in facet passed template should be visible
     SearchToggleFacetSteps.getDualValueFacet().find('[appCypressData="template-title"]')
         .should('be.visible');
-    // And dual value facet should include 'false - 343987'
+    // And dual value facet should include 'null - null'
     SearchToggleFacetSteps.getDataTemplateOfDualValueFacet()
-        .should('contain', 'false - 343987');
-    // And single value facet should include 'Should I filter? - 666'
-    SearchToggleFacetSteps.getDataTemplateOfSingleValueFacet()
-        .should('contain', 'Should I filter? - 666');
+        .should('contain', 'null - null');
```

Comment:
I don't get this. Can you clarify it a bit?

Reply:
I made the facet to fall to default null option if there is no selection passed to the model.
I left the values in the template and they resulted in 'null - null'.
I now added a ngIf condition to the template and check for visibility in the test.